### PR TITLE
Adobe.Acrobat.Reader.64-bit - Changed installer to multi language installer

### PR DESCRIPTION
**Pull request by krystofkrticka**
Changed installer to MUI version so it detects OS language and installs correct Adobe Reader in correct language

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40297)

### CODE REVIEW

**Comment by krystofkrticka** [comment]
What how can package suddenly become not compliant with manifest 1.0 when you change link.

**Comment by OfficialEsco** [comment]
`Manifest Error: Schema validation failed. Error context: <root>[Installers][0][InstallerSha256] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'.  File: Adobe.Acrobat.Reader.64-bit.installer.yaml Manifest Error: Failed to process field. Field: InstallerSha256 Manifest Error: Required field missing. Field: InstallerSha256`

Also the MUI installer got a different PackageName which is why it turned into different locales

**Comment by krystofkrticka** [comment]
You mean that MUI Reader 64 identifies as Acrobat not Acrobat Reader?

**Comment by OfficialEsco** [comment]
IIRC the DisplayName is `Acrobat Reader MUI`, so you could make a new PackageID for it

**Comment by krystofkrticka** [comment]
> IIRC the DisplayName is `Acrobat Reader MUI`, so you could make a new PackageID for it

Well in windows I see this
image

**Comment by krystofkrticka** [comment]
@OfficialEsco I did same change to 32 bit reader and that was just fine.

**Comment by krystofkrticka** [comment]
> > IIRC the DisplayName is `Acrobat Reader MUI`, so you could make a new PackageID for it
> 
> Well that's weird in Programs and Features I see this (Installed using MUI x64 MUI installer) image

just to double check I installed also just installer for one specific language and name is exactly the same

**Comment by krystofkrticka** [comment]
> > IIRC the DisplayName is `Acrobat Reader MUI`, so you could make a new PackageID for it
> 
> Well that's weird in Programs and Features I see this (Installed using MUI x64 MUI installer) image

just to double check I installed also just installer for one specific language and name is exactly the same

**Comment by krystofkrticka** [comment]
@OfficialEsco I think I found what was the issue I made typo in checksum

File: manifests/a/Adobe/Acrobat/Reader/64-bit/21.007.20099/Adobe.Acrobat.Reader.64-bit.installer.yaml [hunk 1]
```diff
@@ -22,23 +22,7 @@ FileExtensions:
 Installers:
 - InstallerLocale: en-US
   Architecture: x64
```
Comment (OfficialEsco) [inline manifests/a/Adobe/Acrobat/Reader/64-bit/21.007.20099/Adobe.Acrobat.Reader.64-bit.installer.yaml:24]:
```suggestion
- Architecture: x64
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Yeah i can also see `Adobe Acrobat DC`.. so i guess this works fine

**Review by OfficialEsco — APPROVED** [review]

**Comment by OfficialEsco** [comment]
I just stumbled over this thread https://www.reddit.com/r/Windows10/comments/s1qfhv/what_the_heck_am_i_doing_wrong_here/

**Comment by krystofkrticka** [comment]
@OfficialEsco I know this happens to me all the when I try to update LibreOffice using winget.

**Comment by krystofkrticka** [comment]
Today this happened when I tried to update [UltraVNC 1.3.6.0](https://github.com/microsoft/winget-pkgs/tree/master/manifests/u/uvncbvba/UltraVnc/1.3.6.0) on one computer which I have at home.

**Comment by OfficialEsco** [comment]
Acrobat Reader is marked as exe, this means its probably unpacking another installer which possibly is a MSI, we can fix and experiment with this once winget-cli gets support for `AppsAndFeaturesEntry` included in the [1.1.0 Schema](https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json)

LibreOffice is marked as MSI, so i believe you have another language than en-US installed? You can see this in the log. This can be fixed pretty easily i believe

Currently we only have the UltraVNC MSI installer because the EXE prompts for Driver installation, i would bet your have the EXE version installed

**Comment by krystofkrticka** [comment]
> Acrobat Reader is marked as exe, this means its probably unpacking another installer which possibly is a MSI, we can fix and experiment with this once winget-cli gets support for `AppsAndFeaturesEntry` included in the [1.1.0 Schema](https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json)

Yes it’s adobes self extracting archive even wingetcreate recognizes that exe as adobe self extracting archive. You can check this by opening adobes exe in 7-zip or something like that.

**Comment by krystofkrticka** [comment]
> LibreOffice is marked as MSI, so i believe you have another language than en-US installed? You can see this in the log. This can be fixed pretty easily i believe
Even though my primary language is Czech I prefer to use my computer in **English US** and it still happens windows is installed from en-us iso.

**Comment by OfficialEsco** [comment]
I'm updating LibreOffice now and removing the en-US locale which should fix the issue

**Comment by krystofkrticka** [comment]
> I'm updating LibreOffice now and removing the en-US locale which should fix the issue

Oh I didn’t know that specifying language could cause issues once I get to my computer I’ll update packages which I added and remove language tag from packages which have one installer for multiple languages.

**Comment by krystofkrticka** [comment]
> Currently we only have the UltraVNC MSI installer because the EXE prompts for Driver installation, i would bet your have the EXE version installed

Possibility of this is quite high. Since many apps I installed manually, if they weren’t in repos of other package manager, before I started migrating to winget.

**Comment by OfficialEsco** [comment]
> > I'm updating LibreOffice now and removing the en-US locale which should fix the issue
> 
> Oh I didn’t know that specifying language could cause issues once I get to my computer I’ll update packages which I added and remove language tag from packages which have one installer for multiple languages.

Yeah we realized this after time, as most programs don't have a language switch, from now on only links which contain a language code is marked with the locale

> > Currently we only have the UltraVNC MSI installer because the EXE prompts for Driver installation, i would bet your have the EXE version installed
> 
> Possibility of this is quite high. Since many apps I installed manually, if they weren’t in repos of other package manager, before I started migrating to winget.

Yeah, UltraVNC only advertises the EXE, so basically anyone that installed it outside of winget will have the exe, which is why i went for the EXE DisplayVersion instead of the MSI DisplayVersion since its weird, unfortunately we can't hide the update for the exe users..

**Comment by krystofkrticka** [comment]
> Yeah, UltraVNC only advertises the EXE, so basically anyone that installed it outside of winget will have the exe, which is why i went for the EXE DisplayVersion instead of the MSI DisplayVersion since its weird, unfortunately we can't hide the update for the exe users..

Is there a simple way to see how some applications were installed?

**Comment by OfficialEsco** [comment]
You can find the ProductCode via RevoUninstaller
or if you got this reg string `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Ultravnc2_is1`
ProductCode: Ultravnc2_is1

**Comment by OfficialEsco** [comment]
Interesting, imma have to look more into Bulk Crap Uninstaller vs RevoUninstaller since its FOSS, you can see tons of tasty info on the first page, Registry Key and ProductCode both work as ProductCodes 
VirtualBoxVM_65JD1XuNUM

**Comment by krystofkrticka** [comment]
Oh I somehow managed to miss this column. Because yes it shows overwhelming amount of info even for advanced user, but it’s better to have this info rather than not. Thanks

**Pull request merged**